optaplanner-core/src/test/java/org/optaplanner/core/impl/heuristic/selector/common/nearby/LinearDistributionNearbyRandomTest.java

```diff
@@ -34,33 +33,33 @@ public void sizeMaximumTooLow() {
 
     @Test
     public void nextInt() {
-        Random random = mock(Random.class);
+        Random random = new TestRandom(
+                0.0,
+                2.0 / 100.0,
+                2.0 / 100.0 + 2.0 / 100.0 + 2.0 / 10000.0,
+                2.0 / 100.0 + 2.0 / 100.0 + 2.0 / 10000.0 + 2.0 / 100.0 + 4.0 / 10000.0,
+                0.0,
+                2.0 / 10.0);
```

Comment:
(nitpick) any reason why we're doing these all at ones and not per method like in the other code?

This might be easier to read and maintain:
```
assertThat(nearbyRandom.nextInt(new TestRandom(0.0), 500)).isEqualTo(0);
assertThat(nearbyRandom.nextInt(new TestRandom(2.0 / 100.0), 500)).isEqualTo(1);
...
```